drivers/azure/azure.go

```diff
@@ -233,6 +257,11 @@ func (d *Driver) SetConfigFromFlags(fl drivers.DriverOptions) error {
 	d.NoPublicIP = fl.Bool(flAzureNoPublicIP)
 	d.StaticPublicIP = fl.Bool(flAzureStaticPublicIP)
 	d.DockerPort = fl.Int(flAzureDockerPort)
+	d.OS = fl.String(flAzureOS)
+	if d.OS == "windows" {
+		// Open WinRM port
+		d.OpenPorts = append(d.OpenPorts, "5986")
```

Comment:
I suggest doing this in `getSecurityRules` method. https://github.com/docker/machine/blob/9718d6c266a767d09cc7521370d5a49c7307d596/drivers/azure/util.go#L73 so that you can give it a proper name